src/Tasks/Common/Resources/Strings.resx

```diff
@@ -436,4 +436,20 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1080: A PackageReference to Microsoft.AspNetCore.App is not necessary when targeting .NET Core 3.0 or higher. If Microsoft.NET.Sdk.Web is used, the shared framework will be referenced automatically. Otherwise, the PackageReference should be replaced with a FrameworkReference.</value>
     <comment>{StrBegin="NETSDK1080: "}</comment>
   </data>
-</root>
+  <data name="UnresolvedTargetingPack" xml:space="preserve">
+    <value>NETSDK1081: The targeting pack for {0} was not found. You may be able to resolve this by running a NuGet restore on the project.</value>
+    <comment>{StrBegin="NETSDK1081: "}</comment>
+  </data>
+  <data name="NoAppHostAvailable" xml:space="preserve">
+    <value>NETSDK1084: There was no app host for available for the specified RuntimeIdentifier '{0}'.</value>
```

Comment:
```suggestion
    <value>NETSDK1084: There was no application host available for the specified RuntimeIdentifier '{0}'.</value>
```

Reply:
In errors I think we should use `application host` or `apphost` as a shortened form.

Either way, need to remove the extraneous `for` in the sentence.

Reply:
Also, should we use present tense `There is no...` instead?